pandas/tests/indexing/test_loc.py

```diff
@@ -888,22 +888,6 @@ def test_identity_slice_returns_new_object(self):
         original_series[:3] = [7, 8, 9]
         assert all(sliced_series[:3] == [7, 8, 9])
 
-    def test_loc_copy_vs_view(self):
```

Comment:
see https://github.com/pandas-dev/pandas/issues/15631#issuecomment-699074229

Reply:
umm pls revert this test.